src/core/physics/PhysicsRaycastResult.js

```diff
@@ -0,0 +1,44 @@
+
+define(["lib/classy",
+        "lib/three"
+    ], function(Class, THREE) {
+
+/**
+    PhysicsWorld raycast result object
+    @class PhysicsRaycastResult
+    @constructor
+*/
+var PhysicsRaycastResult = Class.$extend(
```

Comment:
Same thing here, probably just nicer to use the properties in code. Extra function calls to get a public properties is just overhead. If the functions want to return a clone to assure the original is not modified, or do some calcs priori to returnining a value then its needed. In this case the "public" property should be prefixed with `_ `. This is kind of the standard way of telling outside code "you should not touch this, use a func instead" (also the property is left undocumented ofc).